### PR TITLE
Update intallation configs

### PR DESCRIPTION
**Pull request by rsevilla87**
### Description

Multiple changes:

- Scale to 24 nodes rather than 27
- Use the kube-burner OCP wrapper to run node-density and variants
- Add cluster-density-v2
- Use smaller infra instance/volumes sizes in some configs 
- Disable workload nodes on control-plane bechmarks
- Remove empty aws_profile, aws_access_key_id and aws_secret_access_key variables. We don't need to specify them, as they are fetched from vault
- Disable non-aws cloud platforms. They are actually commented
- Standarize benchmark and installation config files naming. and update manifest.yaml accordingly.
- Fixes: https://github.com/cloud-bulldozer/airflow-kubernetes/issues/303

[Running playground](http://rsevilla87-update-rosa-config-airflow.apps.sailplane.perf.lab.eng.rdu2.redhat.com/home) if you wan't to take a look

Signed-off-by: Raul Sevilla <rsevilla@redhat.com>

### CODE REVIEW

**Comment by krishvoor** [comment]
Hi @rsevilla87  👋 
Was looking at the failed DAG Unit Tests. Can you update the   `assert hits`
from this file `tests/openshift_nightlies/util/test_manifest.py`

File: dags/openshift_nightlies/config/benchmarks/data-plane.json [hunk 1]
```diff
@@ -26,12 +26,12 @@
...
         {
-            "name": "scale-27",
+            "name": "scale-24",
             "workload": "scale-perf",
             "command": "./run_scale_fromgit.sh",
             "env": {
-                "SCALE": "27",
-                "METADATA_COLLECTION": "true",
+                "SCALE": "24",
+                "METADATA_COLLECTION": "false",
```
Comment (jtaleric) [inline dags/openshift_nightlies/config/benchmarks/data-plane.json:34]: curious about the flip there?

File: dags/openshift_nightlies/config/benchmarks/large-control-plane-mgs.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            "workload": "kube-burner",
+            "trigger_rule": "all_done",
+            "custom_cmd": "kube-burner ocp node-density --uuid=${UUID} --pods-per-node=245 --timeout=2h --pod-ready-threshold=5s --es-server=${ES_SERVER} --es-index=ripsaw-kube-burner"
+        },
+        {
+            "name": "cluster-density",
+            "workload": "kube-burner",
+            "trigger_rule": "all_done",
+            "command": "./run.sh",
+            "env": {
```
Comment (jtaleric) [inline dags/openshift_nightlies/config/benchmarks/large-control-plane-mgs.json:14]: Is this still needed with the wrapper? I think kube-burner will set most of these (not read from them)

https://github.com/cloud-bulldozer/kube-burner/blob/master/pkg/workloads/cluster-density.go#L43-L48

File: dags/openshift_nightlies/config/benchmarks/medium-control-plane-mgs.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
+{
+    "benchmarks": [
+        {
+            "name": "node-density",
+            "workload": "kube-burner",
+            "trigger_rule": "all_done",
+            "custom_cmd": "kube-burner ocp node-density --uuid=${UUID} --pods-per-node=245 --timeout=2h --pod-ready-threshold=5s --es-server=${ES_SERVER} --es-index=ripsaw-kube-burner"
```
Comment (jtaleric) [inline dags/openshift_nightlies/config/benchmarks/medium-control-plane-mgs.json:7]: `5s` seems like a low threshold?

File: dags/openshift_nightlies/config/benchmarks/large-control-plane-mgs.json [hunk 4]
```diff
@@ -0,0 +1,40 @@
+{
+    "benchmarks": [
+        {
+            "name": "node-density",
+            "workload": "kube-burner",
+            "trigger_rule": "all_done",
+            "custom_cmd": "kube-burner ocp node-density --uuid=${UUID} --pods-per-node=245 --timeout=2h --pod-ready-threshold=5s --es-server=${ES_SERVER} --es-index=ripsaw-kube-burner"
```
Comment (jtaleric) [inline dags/openshift_nightlies/config/benchmarks/large-control-plane-mgs.json:7]: `5s` seems like a low threshold.

File: dags/openshift_nightlies/config/benchmarks/large-control-plane.json [hunk 5]
```diff
@@ -30,46 +30,36 @@
                 "WORKLOAD_NODE_ROLE": "workload"
             }
         },
+        {
+            "name": "node-density",
+            "workload": "kube-burner",
+            "trigger_rule": "all_done",
+            "custom_cmd": "kube-burner ocp node-density --uuid=${UUID} --pods-per-node=245 --timeout=2h --pod-ready-threshold=5s --es-server=${ES_SERVER} --es-index=ripsaw-kube-burner"
```
Comment (jtaleric) [inline dags/openshift_nightlies/config/benchmarks/large-control-plane.json:37]: `5s` seems like a low threshold. (repeating this just to track it)

File: dags/openshift_nightlies/config/install/aws/sdn-small.json [hunk 6]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (jtaleric) [inline dags/openshift_nightlies/config/install/aws/sdn-small.json:1]: Can we just get rid of OpenShiftSDN dags?

File: dags/openshift_nightlies/config/benchmarks/data-plane.json (hunk 1 above)
Comment (rsevilla87) [reply dags/openshift_nightlies/config/benchmarks/data-plane.json:34]: We don't use this metadata (backpack)

File: dags/openshift_nightlies/config/install/aws/sdn-small.json (hunk 6 above)
Comment (rsevilla87) [reply dags/openshift_nightlies/config/install/aws/sdn-small.json]: done

File: dags/openshift_nightlies/config/benchmarks/large-control-plane.json (hunk 5 above)
Comment (rsevilla87) [reply dags/openshift_nightlies/config/benchmarks/large-control-plane.json]: Fixed, I've bumped it to 10s, in our latest and recent results we saw a latency ~8s, so I think that's a good value to detect regressions

File: dags/openshift_nightlies/config/benchmarks/large-control-plane-mgs.json (hunk 4 above)
Comment (rsevilla87) [reply dags/openshift_nightlies/config/benchmarks/large-control-plane-mgs.json]: Fixed, I've bumped it to 10s, in our latest and recent results we saw a latency ~8s, so I think that's a good value to detect regressions

**Comment by afcollins** [comment]
Before toggling off workload node entirely, we should resolve https://github.com/cloud-bulldozer/airflow-kubernetes/issues/303

File: dags/openshift_nightlies/config/benchmarks/large-control-plane-mgs.json (hunk 2 above)
Comment (jtaleric) [reply dags/openshift_nightlies/config/benchmarks/large-control-plane-mgs.json:14]: ack

File: dags/openshift_nightlies/config/benchmarks/data-plane.json (hunk 1 above)
Comment (jtaleric) [reply dags/openshift_nightlies/config/benchmarks/data-plane.json:34]: ack

**Review by jtaleric — APPROVED** [review]
Thanks @rsevilla87 lgtm!

**Pull request merged**